src/variables.scss

```diff
@@ -15,6 +15,9 @@ $hf-box-shadow-header: 0 1px 2px 0 rgba(0, 0, 0, 0.1);
 /* Fonts */
 $hf-font-family-sans-serif: 'roboto', sans-serif;
 $hf-font-size-base: 14px;
+$hf-font-size-title: $hf-font-size-base*2;
+
+$hf-line-height-title: 1.32;
```

Comment:
and advice will be to calculate the line-height out of the font base like bootstrap.